DQM/SiStripMonitorClient/plugins/SiStripAnalyser.cc

```diff
@@ -71,17 +71,15 @@ void SiStripAnalyser::beginRun(edm::Run const& run, edm::EventSetup const& eSetu
   edm::LogInfo("SiStripAnalyser") << "SiStripAnalyser:: Begining of Run";
 
   // Check latest Fed cabling and create TrackerMapCreator
-  unsigned long long const cacheID = eSetup.get<SiStripFedCablingRcd>().cacheIdentifier();
-  if (m_cacheID_ != cacheID) {
-    m_cacheID_ = cacheID;
+  if (fedCablingWatcher_.check(eSetup)) {
     edm::LogInfo("SiStripAnalyser") << "SiStripAnalyser::beginRun: "
                                     << " Change in Cabling, recrated TrackerMap";
-    if (!actionExecutor_.readTkMapConfiguration(eSetup)) {
+    detCabling_ = &eSetup.getData(detCablingToken_);
+    if (!actionExecutor_.readTkMapConfiguration(
+            detCabling_, &eSetup.getData(tkDetMapTokenER_), &eSetup.getData(tTopoTokenER_))) {
```

Comment:
The token was declared to be consumed in `EndRun`, but is being used in `beginRun`.